Makefile

```diff
@@ -163,10 +163,15 @@ build-dirs:
 	@mkdir -p bin/$(ARCH)
 	@mkdir -p .go/src/$(PKG) .go/pkg .go/bin .go/std/$(ARCH)
 
-clean: container-clean bin-clean
+clean: container-clean bin-clean  ## cleans the repository's temporary files
 
 container-clean:
 	rm -rf .container-* .dockerfile-* .push-*
 
 bin-clean:
 	rm -rf .go bin
+
+help:
+	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
+
+.DEFAULT_GOAL := help
```

Comment:
I don't think we should change the default.  That's surprising.  plain `make` almost always mean `make all` which means "build the binaries".